lib/parsers/srtEntries.js

```diff
@@ -1,93 +1,105 @@
-/* eslint-disable complexity */
 const { timecodeToMicroseconds } = require('../shared/utils');
+const {
+  potentialIndexRegex,
+  potentialTimecodeRegex,
+  potentialSrtBlockRegex,
+  invalidFirstEntryRegex,
+  validTimecodeRegex,
+  emptyEntryRegex,
+  validEntryRegexGroups,
+  strictTimestampRegex,
+} = require('./srtEntriesRegex');
+
+/**
+ * Functions for parseSrtEntries
+ */
+function standardizeTimestamp(timestamp) {
+  if (strictTimestampRegex.test(timestamp)) return timestamp;
+  return timestamp.replace(/[^\d]+/g, ':').replace(/:(?=\d{3})/, ','); // HH:MM:SS,mmm format
+}
+
+function pushInvalidEntry(acc, cur, options) {
```

Comment:
let's try to keep the invalid entry as an object so it is easy to identify the different pieces: `{idx, timecode, text}`.  Any missing values can be `undefined`